lib/rules/no-warning-comments.js

```diff
@@ -141,12 +141,18 @@ module.exports = {
 
             const matches = commentContainsWarningTerm(node.value);
 
-            matches.forEach(matchedTerm => {
+            matches.forEach(({ type, comment }) => {
+                let commentToDisplay = comment.trim();
+
+                if (commentToDisplay.length >= 30) {
```

Comment:
It would be nice to declare a constant for `30`.

Reply:
cool